src/Traits/ApiKeyTrait.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kanvas\Sdk\Traits;
+
+use Canvas\Http\Exception\InternalServerErrorException;
+
+/**
+ * Trait ResponseTrait.
+ *
+ * @package Canvas\Traits
+ *
+ * @property Users $user
+ * @property Config $config
+ * @property Request $request
+ * @property Auth $auth
+ * @property \Phalcon\Di $di
+ *
+ */
+trait ApiKeyTrait
+{
+    /**
+     * Validate if KANVAS_SDK_API_KEY is set. If set then return the value.
+     * @return string
+     */
+    private function validateSdkKey(): string
+    {
+        if (empty(getenv('KANVAS_SDK_API_KEY'))) {
+            throw new InternalServerErrorException('App needs to set KANVAS_SDK_API_KEY environmental variables to Run. Please review your enviorment variables.');
```

Comment:
@rwhite27 then change it to getSdkKey , instead of validate since the get will validate , and throw a runtimeexecption from php or a error exection from PHP dont use the kanvas exectpion ;)

Reply:
change it to a protected function to overwrite if needed